Maple2.Model/Game/IFieldProperty.cs

```diff
@@ -17,7 +17,7 @@ public class FieldPropertyGravity : IFieldProperty, IByteDeserializable {
     public FieldPropertyGravity(float gravity) {
         Gravity = gravity;
     }
-
+    
```

Comment:
```suggestion

```

Reply:
so you want no space in between constructor and the function WriteTo?